### PR TITLE
Vega node f password

### PR DESCRIPTION
**Pull request by vega2016**
last changes for forgot password and nodemailer notifications

### CODE REVIEW

File: Code/api/services/mailing.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+
+var headers = {
+    accountPinRecovery: "0xAccountPinRecovery"
+},
+    pinRecoveryMailExpiry = 1000 * 3600 * 1,
+    SMTP_config = {
+        service: "Gmail",
+        auth: {
+            user: 'vvega019@fiu.edu',
```
Comment (tmoor039) [inline Code/api/services/mailing.js:14]: your email cannot be hardcoded here ....
use the vipmailer            auth: {
                    user: 'fiuvipmailer@gmail.com',
                    pass: 'vipadmin123'
                }

UserRotutes.js

File: Code/api/services/mailing.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            pass: 'Vega2016-'
```
Comment (tmoor039) [inline Code/api/services/mailing.js:15]: now everyone knows your gmail account password...

File: Code/server.js [hunk 3]
```diff
@@ -44,16 +46,52 @@ var userRoutes = require('./api/routes/userRoutes')(app, express);
...
+//TODO: Make sure userRoutes works... Come back to change this.
+app.use('/userapi', userRoutes);
+app.use('/support', supportRoutes);
+
+
+app.post('/register-project-proposals',function(req,res){
+	var SMTP_config = {
+        service: "Gmail",
+        auth: {
+            user: 'vvega019@fiu.edu',
```
Comment (tmoor039) [inline Code/server.js:64]: same thing...

```
       auth: {
                user: 'fiuvipmailer@gmail.com',
                pass: 'vipadmin123'
            }
```

File: Code/server.js [hunk 4]
```diff
@@ -44,16 +46,52 @@ var userRoutes = require('./api/routes/userRoutes')(app, express);
...
+            pass: 'Vega2016-'
+        }
+    };
+         // create reusable transporter object using the default SMTP transport
+	//var transporter = nodemailer.createTransport('smtps://visualnet2008@gmail.com:20001142h@smtp.gmail.com');
+	var transporter = nodemailer	.createTransport("SMTP", SMTP_config);
+	// setup e-mail data with unicode symbols
+	var mailOptions = {
+	    from: '"Victoriano Vega" <visualnet2008@gmail.com>', // sender address
```
Comment (tmoor039) [inline Code/server.js:73]: this cannot be hard coded, it is from FIU VIP

File: Code/server.js [hunk 5]
```diff
@@ -44,16 +46,52 @@ var userRoutes = require('./api/routes/userRoutes')(app, express);
...
+	    to: 'vvega019@fiu.edu', // list of receivers
```
Comment (tmoor039) [inline Code/server.js:74]: who is the receiver?

File: Code/webapp/features/project-proposals/ProjectProposalController.js [hunk 6]
```diff
@@ -176,4 +177,139 @@ angular.module('ProjectProposalController', ['ProjectProposalService'])
               vm.disciplines.push(majors);
             }
         };
-    });
\ No newline at end of file
+
```
Comment (tmoor039) [inline Code/webapp/features/project-proposals/ProjectProposalController.js:180]: confirm with @avill152  that proposal need these changes

**Comment by tmoor039** [comment]
try to fix if you can... there is a code freeze but this needs to get merged so get rid of hard coded email and should be good

**Comment by avill152** [comment]
Needs to merge in master

**Pull request closed without merging**